docs/examples/visualizer/index.html

```diff
@@ -0,0 +1,550 @@
+<html>
+<!-- A sample application showing integration of D3 into CM-Well using the file hosting capabilities of CM-Well -->
+<!-- Debug with python -m SimpleHTTPServer 8000 -->
+<!-- Post to CM-Well with 
+    - curl -X POST <env>/exercise/files/visualizer.html -H "X-CM-WELL-TYPE: FILE" -H "Content-Type: text/html" -ndex.html"  
+        for the main file and
+    - curl -X POST <env>/exercise/files/d3.min.js -H "X-CM-WELL-TYPE: FILE" -H "Content-Type: application/javascript" --data-binary @"d3.min.js" 
+        for d3
+-->
+
+<head>
+    <title>Stick and ball</title>
```

Comment:
Title should be more meaningful, e.g. "CM-Well Data Visualizer App POC"

Reply:
Done.